social_dilemmas/envs/pettingzoo_env.py

```diff
@@ -0,0 +1,72 @@
+from gym.utils import EzPickle
+from pettingzoo.utils import wrappers
+from pettingzoo.utils.conversions import from_parallel_wrapper
+from pettingzoo.utils.env import ParallelEnv
+
+from social_dilemmas.envs.env_creator import get_env_creator
+
+MAX_CYCLES = 1000
+
+
+def parallel_env(max_cycles=MAX_CYCLES, **ssd_args):
+    return _parallel_env(max_cycles, **ssd_args)
+
+
+def raw_env(max_cycles=MAX_CYCLES, **ssd_args):
+    return from_parallel_wrapper(parallel_env(max_cycles, **ssd_args))
+
+
+def env(max_cycles=MAX_CYCLES, **ssd_args):
+    aec_env = raw_env(max_cycles, **ssd_args)
+    aec_env = wrappers.CaptureStdoutWrapper(aec_env)
+    aec_env = wrappers.AssertOutOfBoundsWrapper(aec_env)
+    aec_env = wrappers.OrderEnforcingWrapper(aec_env)
+    return aec_env
+
+
+class ssd_parallel_env(ParallelEnv):
+    def __init__(self, env, max_cycles):
+        self.ssd_env = env
+        self.max_cycles = max_cycles
+        self.possible_agents = list(self.ssd_env.agents.keys())
```

Comment:
Might be  good to sort this, or check that the order here is reasonably deterministic.